scripts/build_docs_site.sh

```diff
@@ -23,7 +23,16 @@ md_to_adoc() {
     sed -i.bak 's/^##/#/' $tmpfile
 
     # Changes markdown file interlink extensions to .html so they continue to work when the site is rendered.
-    sed -i.bak 's/\.md)/.html)/' $tmpfile
+    sed -i.bak 's/\.md\([)#]\)/.html\1/g' $tmpfile
+
+    # Changes all dashes to underscores inside the anchors.
+    sed -i.bak -e ':loop' -e 's/\(\]([^)]*#[^)]*\)-\([^)]*)\)/\1_\2/g' -e 't loop' $tmpfile
```

Comment:
This is some black magic right here
![giphy](https://user-images.githubusercontent.com/12886084/77442605-3280be00-6df3-11ea-8950-a671e56c3a18.gif)